packages/cache-url/lib/AmpCurlUrlGenerator.js

```diff
@@ -138,42 +143,10 @@ function constructFallbackCurlsCacheDomain_(domain) {
  */
 function sha256_(str) {
   if (typeof window !== 'undefined') {
-    // Transform the string into an arraybuffer.
-    const buffer = new TextEncoder('utf-8').encode(str);
-    return crypto.subtle.digest('SHA-256', buffer).then((hash) => {
-      return hex_(hash);
-    });
+    return ampCurlUrlBrowserSha256(str);
```

Comment:
Can we throw a comment in here for future authors explaining that only one of these imports remains per build?